tests/client/disco/components/TestAddon.js

```diff
@@ -224,10 +225,28 @@ describe('<Addon />', () => {
       assert.ok(themeAction.calledWith(themeImage, THEME_RESET_PREVIEW));
     });
 
-    it('runs preventDefault onClick', () => {
+    it('installs a theme when the theme image is clicked', () => {
       const preventDefault = sinon.stub();
-      Simulate.click(themeImage, {preventDefault});
+      const installTheme = sinon.stub();
+      const data = { ...result, type: THEME_TYPE, themeAction,
```

Comment:
I don't think there should be spaces in the parens here. I find the indentation a bit weird, too.